ec2/index.js

```diff
@@ -1,12 +1,13 @@
 var express = require('express');
 var app = express();
 var bodyParser = require('body-parser');
+var jwt = require('jsonwebtoken');
 
 app.use(bodyParser.urlencoded({ extended: false }))
 app.use(bodyParser.json())
 
 const PORT = 4000;
-const api_key = require('./.keys/api-key.js')
+const private_key = require('./.keys/private-key.js')
```

Comment:
this does not solve the issue that you are still going to have to put this file manually